### ISSUE TITLE
take name of current open file from `usethis::compute_name()`

### ISSUE THREAD

**Issue by maxheld83** [dataheld/elf#2]
```
  # TODO take 
```

https://github.com/dataheld/elf/blob/a2c80907d1719d66ad3a0cb3aaf3d3af7bd49d39/R/examples.R#L29

**Comment by maxheld83** [comment]
blocked by https://github.com/r-lib/usethis/issues/2019

**Comment by maxheld83** [comment]
closing in favor of #4
